keps/2724-topology-aware-scheduling/README.md

```diff
@@ -636,6 +651,9 @@ if it had the `podset-preferred-topology` annotation pointing to the lowest
 topology level defined in the Topology referenced by the selected TAS flavor.
 We call it "implicit default" as the annotation isn't persisted.
 
+To change the implicit default from `podset-preferred-topology` to `podset-unconstrained-topology`
+a user can switch on the `TASImplicitDefaultUnconstrained` feature gate.
```

Comment:
Please add somewhere a note that the feature gate is "deprecated" and only for experimentation to collect user feedback. If users find this configuration valuable we will introduce a dedicated TAS configuration.

Reply:
Done